salt/lax/config/etc-init-uwsgi-bot-lax-adaptor.conf

```diff
@@ -0,0 +1,5 @@
+description "uWSGI server"
+start on (local-filesystems and net-device-up IFACE!=lo)
+respawn
+env LANG=en_US.UTF-8
+exec /opt/bot-lax-adaptor/venv/bin/uwsgi --die-on-term --ini /opt/bot-lax-adaptor/uwsgi.ini
```

Comment:
we can add New Relic later. I think this doesn't need to run as a particular user, right?